src/components/ActivityCollector/createLinkClick.js

```diff
@@ -50,7 +50,7 @@ export default (window, config) => {
     }
 
     if (isValidLink) {
-      event.documentUnloading();
+      event.documentMayUnload();
```

Comment:
Now that `event.documentUnloading()` is called for every link, regardless of whether the document is actually unloading, I renamed it to `event.documentMayUnload()`.